blocks/category-carousel-block/_index.scss

```diff
@@ -0,0 +1,15 @@
+@use "@wpmedia/arc-themes-components/scss";
+
+.b-category-carousel {
+	&__title {
```

Comment:
Why not use the `<Paragraph>` component as that has truncation ability built in?

Reply:
It didn't feel semantically like a <p> to me.  It's a title of the category and not a descriptive element.

Reply:
In that case we should use a heading if it's a title

Reply:
Fixed in latest